scalding-quotation/src/main/scala/com/twitter/scalding/quotation/Projection.scala

```diff
@@ -0,0 +1,143 @@
+package com.twitter.scalding.quotation
+
+sealed trait Projection {
+  def andThen(prop: String, tpe: String): Projection
```

Comment:
I also wonder if we should use some `AnyVal` wrappers here instead of `String`. Something like:
```scala
case class Accessor(asString: String) extends AnyVal
case class TypeName(asString: String) extends AnyVal
```

Reply:
done